presto-hive/src/main/java/io/prestosql/plugin/hive/util/Statistics.java

```diff
@@ -424,9 +420,6 @@ else if (type.equals(DOUBLE) || type.equals(REAL)) {
         else if (type.equals(DATE)) {
             result.setDateStatistics(new DateStatistics(getDateValue(session, type, min), getDateValue(session, type, max)));
         }
-        else if (type.equals(TIMESTAMP_MILLIS)) {
-            result.setIntegerStatistics(new IntegerStatistics(getTimestampValue(min), getTimestampValue(max)));
```

Comment:
lets create a follow up issue and keep a TODO here, so it's easier to impl later

Reply:
done